cppcache/src/SerializationRegistry.cpp

```diff
@@ -287,25 +285,21 @@ void SerializationRegistry::addPdxType(TypeFactoryMethodPdx func) {
   theTypeMap.bindPdxType(func);
 }
 
-void SerializationRegistry::addType(int64_t compId, TypeFactoryMethod func) {
-  theTypeMap.rebind(compId, func);
+void SerializationRegistry::addType(int32_t id, TypeFactoryMethod func) {
+  theTypeMap.rebind(id, func);
```

Comment:
This should probably be `addDataSerializableType(...)` and `rebindDataSerializable`